ui/src/app/pages/data/cohort/cohort-resource-card.spec.tsx

```diff
@@ -1,25 +0,0 @@
-import '@testing-library/jest-dom';
-
-import * as React from 'react';
-
-import { screen } from '@testing-library/react';
-import { CohortResourceCard } from 'app/pages/data/cohort/cohort-resource-card';
-
-import { renderWithRouter } from 'testing/react-test-helpers';
-
-describe('CohortResourceCard', () => {
-  const component = () => {
-    const props = {
-      cohort: {},
-    };
-
-    return renderWithRouter(
-      <CohortResourceCard resource={props} onUpdate={() => {}} />
-    );
-  };
-
-  it('should render', () => {
```

Comment:
this did not seem worthwhile to keep - it would have needed an update due to the signature change